contents/en/examples/index.html

```diff
@@ -1,4 +1,9 @@
 <h1>Examples</h1>
+<p>
+  Looking for more examples?<br>
+  <a href="https://github.com/hajimehoshi/ebiten/tree/main/examples">Examples directory</a> contains all the examples listed on this page and more.<br>
+  <a href="https://github.com/sedyh/awesome-ebiten">The Awesome Ebiten repository</a> links to open source frameworks, libraries and software powered by Ebiten.
+</p>
```

Comment:
* Don't insert new lines
* Don't use `<br>`
* Probably we should change the sentence `Looking for more examples?` to, e.g., `In addition to the below examples,`?